src/game.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './item'
+
+class Game < Item
+  attr_reader :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at, *parent_args)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+    super(parent_args)
+  end
+
+  def can_be_archived?
+    super() and last_played_at < 2
```

Comment:
Awesome work @degisew . 

But a little update can be done on your line 13, to validly meet the can_be_archived conditions, we need to parse the data and minus it from the date.now method. 

I would suggest you import the "require Date" at the top, and then use the logic below to check for the difference between today's date and  last_played_at date in terms of years
 
`super and (Date.today.year - Date.parse(@last_played_at).year > 2)`

In the end, this is a great work, clean and DRY.

Reply:
🚀